glean/generated/pings.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
```

Comment:
Our team recommends not committing the generated code, when possible, as it hides some issues and might be problematic to debug if Glean changes and this is not changed: would you consider removing this?